static/App.js

```diff
@@ -85,7 +86,7 @@ function App() {
         <div>
           <div style={{marginBottom: "1em", paddingLeft: "1em", marginTop: "1em"}}>Search Results</div>
           <div>
-            <PlaceResults placesData={placesData} selectedPlace={selectedPlace} setSelectedPlace={setSelectedPlace} />
+            <PlaceResults placesData={placesData} selectedPlace={selectedPlace} setSelectedPlace={setSelectedPlace} scrollToItemRef={scrollToItemRef} />
```

Comment:
why put this outside of placeresults?

Reply:
If i am understanding this correctly, are you asking why useRef is defined outside of PlaceResults.js? The reason why is because i want to pass scrollToItemRef into Map.js as well.